docs/source/roadmap.md

```diff
@@ -1,19 +1,19 @@
-# _starfish_ 0.1.0 Roadmap
-This document describes the features required for SpaceTx groups to use _starfish_ to analyze their data. These users will generate feedback on _starfish_, enabling work to be tailored towards the most valuable feature set.
+# _starfish_ Roadmap
+This document describes the features and timeline for future work on the _starfish_ package.
 
-The purpose of this document is to arrive at a detailed list of short-term key deliverables that we aim to deliver by July 2019, the targeted release date for 0.1.0. To accomplish this, we first outline our primary use cases. With these to guide us, we then outline [**what we need to build**](#what-we-need-to-build:-starfish-0.1.0), [**what we can build later**](candidates-features-for-0.2.0+), and why.
+The purpose of this document is to arrive at a detailed list of short-term key deliverables that we aim to deliver by June 2019. To accomplish this, we first outline our primary [use cases](#use-cases). With these to guide us, we then define milestones for releases that support these needs [**what we need to build**](#what-we-need-to-build:-starfish-0.1.0), and candidate future use cases which we may consider building [**what we can build later**](candidates-features-for-0.2.0+).
 
-## Primary use cases
-To support SpaceTx and Chan Zuckerberg Biohub benchmarking efforts, _starfish_ needs to support the following use cases.
-
-### Parameter selection on single fields of view
-Image-based transcriptomics workflows are highly dependent upon the tissue, organism, and probes being assayed. Labs typically select image processing parameters by analyzing and optimizing parameter selection in a single field of view. In support of this use case, _starfish_ must enable users to process data for any SpaceTx assay on the local computer of their choosing and interact with visualization tools necessary to evaluate parameter decisions. It must support this functionality for users that are not proficient in python.
+## Use cases
+_starfish_ is being developed to support the data processing and benchmarking needs of the SpaceTx consortium. We have identified a set of use cases from these groups that _starfish_ will support. Feedback from these users will set the course for future work on _starfish_, beyond the milestones identified here.
 
 ### Local processing of data for SpaceTx Users
 Some SpaceTx Assays have moderate processing requirements. Users of these assays would like to be able to use _starfish_ to process multiple fields of view of SpaceTx data on their local machines to enable further parameter tuning and spot-checking of results. To support this, we will define a Python API for running _starfish_ on local machines that processes experiments of arbitrary numbers of fields of view, logs, stores data provenance, is multi-processing enabled. Users will also be able to run _starfish_ pipelines through a command-line interface (CLI).
```

Comment:
```diff
+ Some image-based transcriptomics assays
- Some SpaceTx Assays
```

Reply:
```diff
+ process multiple fields of view of data on their local machines
- process multiple fields of view of SpaceTx data on their local machines
```
It should be "SpaceTx Format" data, but here I think it's simpler just to omit it.